product_docs/docs/pgd/5/planning/limitations.mdx

```diff
@@ -65,15 +65,15 @@ doing so incurs many immediate risks and current limitations:
 
 ## Durability options (Group Commit/CAMO)
 
-There are various limits on how the PGD durability options work; this covers
+There are various limits on how the PGD durability options work. This includes
 Group Commit and CAMO and how they operate with PGD features such as the WAL
```

Comment:
Actually, I wasn't sure about this edit. What does "this" refer to?

Reply:
What about something like this:  
```
These limitations result from the interactions between Group Commit, CAMO, and PGD features such as the WAL decoder and transaction streaming. 
```
@djw-m @jpe442 can you check the meaning is correct?

Reply:
@jpe442 @djw-m If Gaby's edit is good, I can implement it and get this approved. Thanks!

Reply:
These limitations are a product of the interactions between Group Commit and CAMO and how they interact with PGD features such as the [WAL decoder](/pgd/latest/node_management/decoding_worker/) and  [transaction streaming](/pgd/latest/transaction-streaming/).